wormhole-connect/src/components/TokensModal.tsx

```diff
@@ -511,6 +511,16 @@ function TokensModal(props: Props) {
       const isNonzeroBalance = b !== null && b !== '0';
       return isNonzeroBalance;
     });
+
+    // If any of the tokens are native to the chain, only show those.
+    // This is to avoid users inadvertently receiving wrapped versions of the token.
+    if (type === 'dest') {
+      const nativeTokens = filtered.filter((t) => t.nativeChain === chain);
+      if (nativeTokens) {
```

Comment:
```suggestion
      if (nativeTokens.length > 0) {
```

an empty array is truthy

Reply:
fixed